django-verdant/rca/templates/rca/base.html

```diff
@@ -116,6 +116,17 @@
             <p class="chromeframe">You are using an <strong>outdated</strong> browser which this site does not fully support. Please <a href="http://browsehappy.com/">upgrade your browser</a> or <a href="http://www.google.com/chromeframe/?redirect=true">activate Google Chrome Frame</a> to improve your experience.</p>
         <![endif]-->
 
+        <div class="survey-banner" id="#survey-banner">
+            <div class="survey-banner__content">
+                <div class="survey-banner__image-wrapper">
+                    <img class="survey-banner__image" src="{% static "rca/images/amzn100.png" %}" />
+                </div>
+                <p class="survey-banner__text">Give us feedback <span class="survey-banner__dash">- &nbsp;</span><span class="survey-banner__bold-text">receive a £10 Amazon.co.uk Gift Card.</span></p>
```

Comment:
`survey-banner__bold-text` should be a `strong` tag for accessibility,

```suggestion
                <p class="survey-banner__text">Give us feedback <span class="survey-banner__dash">- &nbsp;</span><strong class="survey-banner__bold-text">receive a £10 Amazon.co.uk Gift Card.</strong></p>
```

Reply:
Interesting - does that affect the way it is read out loud?

Reply:
(I have updated)